README.md

```diff
@@ -19,97 +20,149 @@ This WordPress project is intended to provide:
 
-  ```
-  mysql -uroot
-  CREATE DATABASE chappress_dev;
-  GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
-  FLUSH PRIVILEGES;
-  EXIT
-  ```
+# The wp-config file already contains the database information to get started.
+# These are for local development only.
+```
 
-- Download WordPress and run the server.
+- **Install Automated Test Suite**
 
-  ```
-  git clone git@github.com:chapmanu/chap-press.git
-  cd chap-press/public
-  cp -v wp-config.php{-dist,}
+Navigate to project folder root: `/chap-press` and use composer:
 
-  # The wp-config file already contains the database information to get started.
-  # These are for local development only.
+```
+composer install
+./vendor/bin/codecept --version
 
-  php -S localhost:8000
-  ```
+#This will install Codeception for Wordpress.  
+```
 
-  Server should be running in the terminal displaying PHP version, port location, and document root.
+- **Run the Server**
+  
+`php -S localhost:8000`
 
-  Go to your browser and enter the url port:
-  ```
-  http://localhost:8000
-  ```
 
-  Finish the final instructions through the WordPress installation. The WordPress admin panel should be displayed and the user should have full access to developing in a local environment.
+Go to your browser and enter the url port: `http://localhost:8000`
+
+Server should be running in the terminal displaying PHP version, port location, and document root.
+Finish the final instructions through the WordPress installation. The WordPress admin panel should be displayed and the user should have full access to developing in a local environment.
+
+***
 
 ## Debugging
 
-  WordPress comes with specific debug systems designed to simplify the process.
-  The following are meant only for local testing and staging installs.
+WordPress comes with specific debug systems designed to simplify the process.
+The following are meant only for local testing and staging installs.
+
+The following code is already set in the `wp-config.php` file.
+It will log all errors, notices, and warnings to a file called `debug.log` in the wp-content directory.
+It will also hide the errors so they do not interrupt page generation.
+
+<br/>
+
+| Command | Description |
+| --- | --- |
+| `define( 'WP_DEBUG', true );` | Enable WP_DEBUG mode |
+| `define( 'WP_DEBUG_LOG', true );` | Enable Debug logging to `/wp-content/debug.log` |
+| `define( 'WP_DEBUG_DISPLAY', false );` | Disable display of errors and warnings |
+| `@ini_set( 'display_errors', 0 );` | // |
+| `define( 'SCRIPT_DEBUG', true );` | Uses unminified versions of core JS and CSS files |
+
+***
+
+## Automated Testing ###
+
+Please make sure your local development server is running. Tests are executed with 'run' command.
+
+Codeception is executed as:
+
+`./vendor/bin/codecept`
+
+For brevity, you can create an alias that refers to that path:
+
+`#For a permanent alias, add this to your local path environment (e.g. ~/.bash_profile)`
```

Comment:
Here can you just provide command to append the alias line to ~/.bash_profile?

    echo "alias codecept=./vendor/bin/codecept" >> ~/.bash_profile

Or however it's best done.